File: Program 3 Hypotenuse/RightTrig.java
// This program will input the value of two sides of a right triangle and then
// determine the size of the hypotenuse.
// Sara Kendig

import java.util.Scanner;    //import scanner class

public class RightTrig {
    public static void main(String[] args){
        double a;              // side a
        double b;              // side b
        double hyp;            // hypotenuse
        

        Scanner keyboard = new Scanner(System.in);     // create a scanner object

        // get side a
        System.out.println("Please enter first value: ");

        a = keyboard.nextDouble();

        // get side b
        System.out.println("Please enter second value: ");

        b = keyboard.nextDouble();

        // calculate hypotenuse
        hyp = Math.sqrt((a*a)+(b*b));

        // display results
        System.out.println("The sides of the right triangle are " + a + " and " + b);

        System.out.println("The hypotenuse is " + hyp);

        
    }
}

File: Program 3 Hypotenuse/righttrig.cpp
// This program will input the value of two sides of a right triangle and then
// determine the size of the hypotenuse.

// Sara Kendig

#include <iostream>
#include <iomanip>            // Use for setprecision and setw
#include <cmath>              // needed for math functions like sqrt()
using namespace std;

int main()
{
   float a, b;        // the smaller two sides of the triangle
   float hyp;        // the hypotenuse calculated by the program

   cout << setprecision(1) << fixed << showpoint;	// formatted output 

   cout << "Please enter first value: ";
   cin >>   a; // input first value variable
   cout << endl;
   cout << "Please enter second value: ";
   cin >>   b; // input second value variable

   hyp = sqrt(pow(a,2) + pow(b,2));

   cout << endl << "The sides of the right triangle are " << a << " and " << b << endl;

   cout << "The hypotenuse is "  << hyp << endl;

   return 0;
}




